selinuxfs/avc_cache_stats.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux && !noselinux
+// +build linux,!noselinux
+
+package selinuxfs
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type AVCStat struct {
+	Lookups     uint64
+	Hits        uint64
+	Misses      uint64
+	Allocations uint64
+	Reclaims    uint64
+	Frees       uint64
+}
+
+func (fs FS) parseAVCStats() (AVCStat, error) {
```

Comment:
This should probably be an exported function. I don't see any other way to get an `AVCStat` struct returned.

Reply:
true; done